mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -44,4 +44,8 @@
             android:name="flutterEmbedding"
             android:value="2" />
     </application>
+    <uses-permission android:name="android.permission.BLUETOOTH" />
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```

Comment:
Pourquoi est ce que l'on a besoin de la location approximative?

Reply:
C'est un requis de bluetooth depuis Android 6.0 d'avoir ACCESS_FINE_LOCATION or ACCESS_COARSE_LOCATION. Lequel utilisé exactement c'est pas spécifié pour quelle situation, mais ça semble bien marcher avec COARSE.

Reply:
https://stackoverflow.com/questions/33045581/location-needs-to-be-enabled-for-bluetooth-low-energy-scanning-on-android-6-0